contrib/nydus-test/framework/nydusify.py

```diff
@@ -46,7 +49,7 @@ def __init__(self, anchor: NydusAnchor):
         self.cmd = NydusifyParam(self.nydusify_bin)
         self.cmd.nydus_image(self.image_builder).work_dir(self.work_dir)
 
-    def convert(self, source, suffix="_converted", target_ref=None, fs_version=5):
+    def convert(self, source, suffix="_converted", target_ref=None, fs_version=6):
```

Comment:
Let's drop the default value for `fs_version` to make sure that callers always set it properly.

Reply:
Same applies to the `check` method.